lms/envs/common.py

```diff
@@ -1248,9 +1248,6 @@
 PIPELINE_CSS_COMPRESSOR = None
 PIPELINE_JS_COMPRESSOR = 'pipeline.compressors.uglifyjs.UglifyJSCompressor'
 
-# Setting that will only affect the edX version of django-pipeline until our changes are merged upstream
```

Comment:
Reviewers: @robrap and I determined that this setting is no longer necessary. It is not available in django-pipeline.